### PR TITLE
VideoCORS plugin update

### PR DESCRIPTION
**Pull request by bbaudel**
Rename SafariCORS plugins to VideoCORS and add support of Internet Explorer.

### CODE REVIEW

**Comment by sw-team-release-gpsw** [comment]
[CLA assistant check](https://cla.gopro.com/sign) <br/>All committers have signed the CLA.

File: VideoCORS/src/VideoCORS.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ ForgePlugins.VideoCORS.prototype =
 
         var browserName = FORGE.Device.browser.charAt(0).toUpperCase() + FORGE.Device.browser.slice(1);
```
Comment (rroux-gpsw) [inline VideoCORS/src/VideoCORS.js]: Maybe we should add this to FORGE.Device ?

Comment (rroux-gpsw) [reply VideoCORS/src/VideoCORS.js]: Ho forget it, it is just the first char in upper case, browser is already in FORGE.Device

**Review by rroux-gpsw — APPROVED** [review]
LGTM

**Pull request merged**